temporalio/contrib/openai_agents/_temporal_openai_agents.py

```diff
@@ -250,6 +255,20 @@ def init_worker_plugin(self, next: temporalio.worker.Plugin) -> None:
         """Set the next worker plugin"""
         self.next_worker_plugin = next
 
+    @staticmethod
+    def _data_converter(converter: Optional[DataConverter]) -> DataConverter:
+        if converter is None:
+            return DataConverter(payload_converter_class=OpenAIPayloadConverter)
+        elif isinstance(converter.payload_converter, DefaultPayloadConverter):
```

Comment:
```suggestion
        elif converter.payload_converter is PayloadConverter.default:
```

`DefaultPayloadConverter` is uses the word "default" as in "default behavior" not that it is the sole default. It is very common for people to instantiate/extend the `DefaultPayloadConverter`. Alternatively you could:

```suggestion
        elif converter.payload_converter_class is DefaultPayloadConverter:
```